src/extension/src/MsftLinuxPatchExtShim.sh

```diff
@@ -109,8 +107,8 @@ find_python PYTHON
 
 
 if [ -z "$PYTHON" ]; then
-   echo "No Python interpreter found on the box" >&2
-   exit 51 # Not Supported
+   echo "Python is required, but no Python interpreter was found on the box" >&2
+   exit 52
```

Comment:
What's the error code mapping for 52?

Reply:
[Error code mapping for 52 is missing dependency](https://github.com/Azure/azure-marketplace/wiki/Extension-Build-Notes-Best-Practices#error-codes-and-messages-output-to-stderr); error code 51 is unsupported OS.